### PR TITLE
IDE-4075 add options "Dev Studio only" for installer

### PR DESCRIPTION
**Pull request by haoyisun**
IDE-4075 add options "Dev Studio only" for installer

### CODE REVIEW

File: build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml [hunk 1]
```diff
@@ -492,6 +535,27 @@
             <mustExist>0</mustExist>
             <width>30</width>
         </directoryParameter>
+        <choiceParameterGroup>
+            <name>sdk</name>
```
Comment (jtydhr88) [inline build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml]: use cli

File: build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml [hunk 2]
```diff
@@ -492,6 +535,27 @@
...
+            <description>Constain Dev SDK</description>
```
Comment (jtydhr88) [inline build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml]: wrong word spelling
And calling cli tools

File: build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml [hunk 3]
```diff
@@ -492,6 +535,27 @@
...
+            <explanation>If install Dev Studio without any sdk, please choose "Dev Studio only", otherwise default.</explanation>
```
Comment (jtydhr88) [inline build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml]: not sdk, call cli tools

**Comment by jtydhr88** [comment]
thanks Haoyi, see comments and resend, thanks.

**Comment by haoyisun** [comment]
ok

**Pull request closed without merging**